src/qcodes/configuration/qcodesrc.json

```
@@ -78,7 +78,8 @@
         "export_name_elements": ["captured_run_id", "guid"],
         "export_chunked_export_of_large_files_enabled": false,
         "export_chunked_threshold": 1000,
-        "in_memory_cache": true
+        "in_memory_cache": true,
+        "load_from_file": false
```

```suggestion
        "load_from_exported_file": false
```

is perhaps clearer ?